zowe-rest-api-sample-spring/docs/zos-security.md

```diff
@@ -341,16 +347,22 @@ The second `@PreAuthorize` expression `hasSafServiceResourceAccess('RESOURCE', '
 You need following access to be able to develop a REST API service:
 
 - `UPDATE` access to the `SUPERUSER.FILESYS.MOUNT` resource in the UNIXPRIV class
-- `READ` access to `BPX.FILEATTR.PROGCTL` and `BPX.FILEATTR.APF` in the facility class
-- `UPDATE` access to `BPX.SERVER`
+- `READ` access to `BPX.FILEATTR.PROGCTL` and `BPX.FILEATTR.APF` in the FACILITY class
+
+You need following access to be able to run a REST API service:
+
+- `READ` access to `BPX.SERVER` in the FACILITY class to be able to check access to resources
+- `READ` access to `BPX.DAEMON` in the FACILITY class to be able to establish thread-level security
+- nothing special if you want to be able to authenticate users
 
 Commands for CA Top Secret for z/OS:
```

Comment:
We'll see if someone on the team can collect the commands for ACF2.  This is excellent; thanks for documenting!